numpy/core/_add_newdocs.py

```diff
@@ -3114,13 +3114,19 @@
         'C' otherwise. 'K' means match the layout of `a` as closely
         as possible. (Note that this function and :func:`numpy.copy` are very
         similar, but have different default values for their order=
-        arguments.)
+        arguments, and this function always passes sub-classes through.)
 
     See also
     --------
-    numpy.copy
+    numpy.copy : Similar function with different default behavior
     numpy.copyto
 
+    Notes
+    -----
+    This function is the preferred method for creating an array copy.  The
+    function :func:`numpy.copy` is similar, but it defaults to using order 'C',
```

Comment:
From the `numpy.copy` docs, the default is `order='K'` which means *match the layout of a as closely as possible*.

Reply:
Yep, that was a typo.  Thanks for catching that!